docker-compose/s3-credentials.yaml

```diff
@@ -26,6 +27,7 @@ storage:
           secretAccessKey: fakeSecret2
       provider: Other
       isMain: false
+      syncACLGrants: false
```

Comment:
I really like the idea of making this behaviour optional and configuring it with a feature flag. 
Can you please do the following changes to comply with existing chorus feature flags:
- rename to something like `preserveACLGrants`
- move this config parameter from storages config to [features config](https://github.com/clyso/chorus/blob/main/pkg/features/features.go). 
- move config default value from `docker-compose/s3-credentials.yaml` to [pkg/config/config.yaml](https://github.com/clyso/chorus/blob/main/pkg/config/config.yaml#L31-L37).